azext_iot/central/services/enrollment_group.py

```diff
@@ -418,6 +418,14 @@ def verify_x509(
     Verify the primary or secondary x509 certificate of an enrollment group
     by providing a certificate with the signed verification code.
 
+    Please note that if an enrollment group was created with an unverified x509
```

Comment:
If this is intended to be user facing reference doc updates you'll want to update the description in the help module here https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/central/_help.py#L1759

Reply:
@digimaun my intention is to update this http public doc: https://learn.microsoft.com/en-us/cli/azure/iot/central/enrollment-group?view=azure-cli-latest#az-iot-central-enrollment-group-verify-certificate

Is that pulling from _help.py? I was under the assumption that the _help.py was for the cli help command.

Reply:
yes the file paymaun linked is the correct help file you want to edit. What you are doing here is just poking a doc string that affects nothing

Reply:
@vilit1 @digimaun Great, thanks for pointing out the right location. Will push an update soon.